dtcw

```diff
@@ -54,10 +54,14 @@ sdkman=false
 homefolder=false
 java=false
 doJavaCheck=true
-arch=$(uname -m)
+if [ "$(arch)" == "i386" ]; then
+  arch="x86_64"
+else 
+  arch=$(uname -m)
+fi
```

Comment:
Please provide some context within the code why we do this.
Additonal a reference to the ticket where we discussed the problem may help.

Reply:
We had some unexpected behaviour of the `uname` command. I can't remember - it returned "" or "arm64" but the system was in emulated i386 mode. The `arch` command provided better results, but it returns other identifiers.
So, this code helped to solve this issue.

What do you think about it?

Reply:
Damn, Apple doesn't make it easy. Searching the web, I found [a comment on how to detect Apple Silicone on command line](https://stackoverflow.com/a/68148776).

Nevertheless, I don't think this is the right place to make the mapping. We map the `arch` environment variable at https://github.com/docToolchain/docToolchain/blob/6178309da8968c8f99ba1575faacce885677575f/dtcw#L292-L295 again with the goal download the correct JVM.

IMHO it doesn't make sense to map the output of `uname -m` (or `arch`) twice. I would like to provide a clean solution with #1092. 

Does that make sense?